### PR TITLE
added Japanese translation

### PR DESCRIPTION
**Pull request by magiruuvelvet**

### CODE REVIEW

**Comment by kevinpapst** [comment]
Awesome 👍  Is it complete?

**Comment by magiruuvelvet** [comment]
yes all strings translated

File: Resources/translations/AdminLTEBundle.ja.xliff [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file date="2017-01-12T20:00:00Z" source-language="en" target-language="ja" datatype="plaintext" original="none">
```
Comment (kevinpapst) [inline Resources/translations/AdminLTEBundle.ja.xliff]: Can you change the "date" and the original="AdminLTEBundle.en.xliff"

Comment (magiruuvelvet) [reply Resources/translations/AdminLTEBundle.ja.xliff]: done

File: Resources/translations/AdminLTEBundle.en.xliff [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
-    <file date="2017-01-12T20:00:00Z" source-language="en" target-language="en" datatype="plaintext" original="none">
+    <file date="2019-04-10T18:45:00Z" source-language="en" target-language="en" datatype="plaintext" original="AdminLTEBundle.en.xliff">
```
Comment (kevinpapst) [inline Resources/translations/AdminLTEBundle.en.xliff]: Sorry for the misunderstanding, I just meant your file - so please revert this one ;-)

File: Resources/translations/AdminLTEBundle.ja.xliff (hunk 1 above)
Comment (kevinpapst) [reply Resources/translations/AdminLTEBundle.ja.xliff]: so the change needs to be in here in the .ja file

File: Resources/translations/AdminLTEBundle.en.xliff (hunk 2 above)
Comment (magiruuvelvet) [reply Resources/translations/AdminLTEBundle.en.xliff]: i accidentally updated the wrong file, oops

**Comment by kevinpapst** [comment]
And while you are on it, you could Japanese to the list of translations in the README as well: 
https://github.com/kevinpapst/AdminLTEBundle/blob/master/README.md#features

**Comment by magiruuvelvet** [comment]
i added it to the list

**Pull request merged**